public/components/common/search/search.tsx

```diff
@@ -6,9 +6,10 @@
 import '@algolia/autocomplete-theme-classic';
 import {
   EuiBadge,
+  EuiSmallButton,
```

Comment:
Should we replace the existing button import here?

Reply:
No. there are `EuiButton` that were already `size="s"`.